packages/router/tests/ssr_router.rs

```diff
@@ -13,16 +13,16 @@ fn generates_without_error() {
     assert_eq!(out, "<nav>navbar</nav>default<!--placeholder-->");
 }
 
-fn app(cx: Scope) -> Element {
```

Comment:
Commenting this out causes the test to stop compiling. We could add ssr as a dev dependency to the router.